spec/views/admin/content_assets/index.html.erb_spec.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe "admin/content_assets/index", type: :view do
   before(:each) do
     @content_assets = [
-      FactoryBot.create(:content_asset, title: "title 1", alt_text: "hello"),
-      FactoryBot.create(:content_asset, title: "title 2", alt_text: "hello 2 u"),
+      FactoryBot.create(:content_asset, title: "title 1", alt_text: "hello", content_page_id: 1),
+      FactoryBot.create(:content_asset, title: "title 2", alt_text: "hello 2 u", content_page_id: 2),
```

Comment:
If you are not using the content page anywhere in the spec, just let the factory do its job (that is, build an associated object unless directed to do otherwise). So just leave it as:
```ruby
  FactoryBot.create(:content_asset, title: "title 1", alt_text: "hello"),
  FactoryBot.create(:content_asset, title: "title 2", alt_text: "hello 2 u"),
```
The factory will do the rest for you.